docs/pages/index.md

```diff
@@ -10,26 +10,25 @@ title: Home
 
 ### Getting started
 
-- Install [binaries](./run/getting-started/installation.md/#binaries), follow the installation method for [source install](./run/getting-started/installation.md/#build-from-source) or [Docker install](./run/getting-started/installation.md/#docker-installation) to install Lodestar. Or use our [Lodestar Quickstart scripts](https://github.com/ChainSafe/lodestar-quickstart).
+- Install [binaries](./run/getting-started/installation/#binaries), follow the installation method for [source install](./run/getting-started/installation/#build-from-source) or [Docker install](./run/getting-started/installation/#docker-installation) to install Lodestar. Or use our [Lodestar Quickstart scripts](https://github.com/ChainSafe/lodestar-quickstart).
 - Use [Lodestar libraries](./supporting-libraries/index.md) in your next Ethereum Typescript project.
 - Run a beacon node on [mainnet or a public testnet](./run/beacon-management/starting-a-node.md).
 - Utilize the whole stack by [starting a local testnet](./contribution/advanced-topics/setting-up-a-testnet.md).
-- View the Lodestar [CLI commands and options](./run/beacon-management/beacon-cli.md)
+- View the Lodestar Beacon [CLI commands and options](./run/beacon-management/beacon-cli.md)
 - Prospective contributors can read the [contributing section](./contribution/getting-started.md) to understand how we develop and test on Lodestar.
-- If you have questions [submit an issue](https://github.com/ChainSafe/lodestar/issues/new) or join us on [Discord](https://discord.gg/yjyvFRP)!
+- If you have questions [submit an issue](https://github.com/ChainSafe/lodestar/issues/new/choose) or join us on [Discord](https://discord.gg/yjyvFRP)!
 - Please note our [security policy](./security.md).
-- Sign up to our [mailing list](https://chainsafe.typeform.com/lodestar) for announcements and any critical information about Lodestar.
 
 ## Specifications
 
 Hardware specifications minimum / recommended, to run the Lodestar client.
 
-|           | Minimum                          | Recommended                       |
-| --------- | -------------------------------- | --------------------------------- |
-| Processor | Intel Core i5–760 or AMD FX-8100 | Intel Core i7–4770 or AMD FX-8310 |
-| Memory    | 4GB RAM                          | 8GB RAM                           |
-| Storage   | 20GB available space SSD         | 100GB available space SSD         |
-| Internet  | Broadband connection             | Broadband connection              |
+|           | Minimum                                | Recommended                            |
```

Comment:
Noting that these new updated specifications are based on recent experiences with running Lodestar and not necessarily based on baseline benchmark data. For CPU minimums, I based it off https://ethereum.org/en/run-a-node/ where they recommend a baseline CPU benchmark score of 6667+ and I picked out CPUs around this benchmark for minimums.